_monitoring-plugins/ad/api.md

```diff
@@ -1974,20 +1975,26 @@ Introduced 1.0
 
 Returns all results for a search query.
 
-To search anomaly results for `grade` greater than 0 for real-time analysis:
+Specify the `only_query_custom_result_index` parameter (default value is `false`) to search both the custom result index and default result index. To search only the custom result index, set this parameter to `true`:
```

Comment:
Seems not so accurate. If user needs to search both custom result index and default result index, they can add custom result index to search result API, like `POST _plugins/_anomaly_detection/detectors/results/_search/<custom_result_index>`, or `POST _plugins/_anomaly_detection/detectors/results/_search/<custom_result_index>?only_query_custom_result_index=false`

Reply:
If user need to search just default result index, just use `POST _plugins/_anomaly_detection/detectors/results/_search/`

Reply:
If user need to search just custom result index, add custom result index and `? only_query_custom_result_index=true`, like this `POST _plugins/_anomaly_detection/detectors/results/_search/<custom_result_index>?only_query_custom_result_index=true`